From: Victhor Foster <victhor.foster@gmail.com>
Date: 2016-12-04 00:06:22 -0300
Subject: uhd: fix build failure on armv6l and armv7l
References:
Upstream: no

Fix for build failure on armv6l and armv7 architectures

---
	lib/convert/CMakeLists.txt |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/lib/convert/CMakeLists.txt	2016-12-04 00:06:14.869529185 -0300
+++ b/lib/convert/CMakeLists.txt	2016-12-04 00:06:02.025634839 -0300
@@ -62,7 +62,7 @@
 
 IF(HAVE_ARM_NEON_H AND (${CMAKE_SIZEOF_VOID_P} EQUAL 4))
     ENABLE_LANGUAGE(ASM)
-
+    SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mfpu=neon")
     LIBUHD_APPEND_SOURCES(
         ${CMAKE_CURRENT_SOURCE_DIR}/convert_with_neon.cpp
         ${CMAKE_CURRENT_SOURCE_DIR}/convert_neon.S
